neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -103,22 +109,13 @@ type Guest struct {
 }
 
 type CPUs struct {
-	// +kubebuilder:validation:Minimum=1
```

Comment:
I had to remove these validations because they only work for integers. Should probably add manual code for that to the controller